### ISSUE TITLE
As a reader of the Open Forest wiki, I can see up-to-date manual accessibility processes, so I understand the approach being used to ensure accessibility

### ISSUE THREAD

**Issue by jstrothman** [USDAForestService/USFS-timber-permitting#201]
Tasks:
- [ ] Review https://github.com/USDAForestService/fs-open-forest/wiki/Manual-accessibility-testing-process for accuracy/current information
- [ ] Consider explicit mention of the need for PDF accessibility testing
- [ ] Make adjustments to current process as needed
- [ ] Make adjustments to wiki page if needed

**Comment by Rebekah-Hernandez** [comment]
@jstrothman After reviewing, our current process is not included on this wiki page. We might need to have a meeting to discuss. I sent you an email with more details. :)

Please include @Dmac26 as well since he's been working with Brian on the pdf manual testing.

**Comment by Rebekah-Hernandez** [comment]
@jstrothman Were decisions made on this issue for what we wanted to keep in the manual compliance testing from the old documentation?

**Comment by jstrothman** [comment]
@Rebekah-Hernandez My understanding is that in your design/dev sync you all will determine your workflow for ensuring accessibility of the website and generated pdfs. Our strong recommendation is that the process includes testing with a screen reader. If you'd like to recommend changes to the process described in the wiki, perhaps those should be presented to the Forest Service team. In any case, the goal is that the wiki is a match for the process actually being used.

**Comment by Rebekah-Hernandez** [comment]
@jstrothman What is the name of the screen reader that you recommend and used during the demo?
